data/ecosystems/c/cardano.toml

```diff
@@ -20,6 +20,8 @@ github_organizations = [
   "https://github.com/CardanoSolutions",
   "https://github.com/input-output-hk",
   "https://github.com/stakhanovite-io",
+  "https://github.com/txpipe",
+  "https://github.com/dcSpark",
```

Comment:
This casts too wide a net because dcSpark is working on a variety of things.  They already have their own ecosystem.